recipes/recipes_emscripten/h5py/patches/configure.patch

```diff
@@ -0,0 +1,65 @@
+From 7aea0dd0b4b86d9c607acc981d8055e55c9685bb Mon Sep 17 00:00:00 2001
+From: ryanking13 <def6488@gmail.com>
+Date: Thu, 30 Jun 2022 05:30:27 +0000
+Subject: [PATCH] Fix feature detection
+
+---
+ setup_configure.py | 26 ++++----------------------
+ 1 file changed, 4 insertions(+), 22 deletions(-)
+
+diff --git a/setup_configure.py b/setup_configure.py
+index c16ddeef..1180d3fa 100644
+--- a/setup_configure.py
++++ b/setup_configure.py
+@@ -206,7 +206,7 @@ class BuildConfig:
+ class HDF5LibWrapper:
+ 
+     def __init__(self, libdirs):
+-        self._load_hdf5_lib(libdirs)
```

Comment:
I have not chased through how, but I think that this patch is breaking all of h5py's auto-detection of what capabilities the wrapped libhdf5 has.